src/components/Button.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class Button extends Component {
+  constructor(props) {
+    super(props);
+    const { content } = this.props;
+    this.classy = content === '0' ? 'btn0' : 'btn';
+    this.orange = ['÷', 'x', '+', '-', '='].includes(content) ? ' orange' : '';
+  }
+
+  render() {
+    const { content } = this.props;
+    return (
+      <p className={this.classy + this.orange}>{ content }</p>
```

Comment:
- [ ] OPTIONAL: The right semantic tag here is `button`. Even though it is an optional comment, you should take it seriously. :pray: